tsdb/db_test.go

```diff
@@ -1280,21 +1280,21 @@ func TestSizeRetention(t *testing.T) {
 	db.opts.MaxBytes = sizeLimit // Set the new db size limit one block smaller that the actual size.
 	testutil.Ok(t, db.reload())  // Reload the db to register the new db size.
 
-	expBlocks := blocks[1:]
+	expBlocks := blocks[3:]
 	actBlocks := db.Blocks()
 	blockSize = int64(prom_testutil.ToFloat64(db.metrics.blocksBytes))
 	walSize, err = db.Head().wal.Size()
 	testutil.Ok(t, err)
-	// Expected size should take into account block size + WAL size
+	// Expected size should take into account block size + WAL size + checkpoint size
 	expSize = blockSize + walSize
```

Comment:
Comments do not match what is done here :P
Additionally, we should update the test to create a checkpoint _somewhere_